### PR TITLE
chore: upgrade husky version to 1.1.1

### PR DESCRIPTION
**Pull request by cmygray**
Proposed changes:
  - use husky@1.1.1

Solves #41

### CODE REVIEW

**Comment by deerawan** [comment]
Thank you @cmygray for this PR

I'm afraid we also need to move this line
https://github.com/deerawan/vscode-dash/blob/051c3a3fdbdc1a720688ed9de37a50252132bf44/package.json#L406

to conform with husky 1.1 standard for example
```
// package.json
{
  "husky": {
    "hooks": {
      "pre-commit": "npm test",
      "pre-push": "npm test",
      "...": "..."
    }
  }
}
```

File: package.json [hunk 1]
```diff
@@ -411,7 +413,7 @@
   "devDependencies": {
     "all-contributors-cli": "^5.4.0",
     "coveralls": "^2.11.4",
-    "husky": "^0.14.3",
+    "husky": "1.1.1",
```
Comment (deerawan) [inline package.json]: Thank you @cmygray for this PR

I'm afraid we also need to move this line
https://github.com/deerawan/vscode-dash/blob/051c3a3fdbdc1a720688ed9de37a50252132bf44/package.json#L406

to conform with husky 1.1 standard for example
```
// package.json
{
  "husky": {
    "hooks": {
      "pre-commit": "npm test",
      "pre-push": "npm test",
      "...": "..."
    }
  }
}
```

**Comment by cmygray** [comment]
Resolved requested changes.

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,3538 @@
+Arguments:
```
Comment (deerawan) [inline yarn-error.log]: should we remove this file since we don't use `yarn` in this project? 😄

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,3066 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (deerawan) [inline yarn.lock]: should we remove this file since we don't use yarn in this project? 😄

File: package.json [hunk 4]
```diff
@@ -421,6 +422,11 @@
     "typescript": "^2.9.0",
     "vscode": "0.10.x"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```
Comment (deerawan) [inline package.json]: 👍

**Comment by cmygray** [comment]
yarn files are removed. thx and sorry for waiting.

**Comment by deerawan** [comment]
@cmygray sorry for late response. Thank you :)

**Pull request merged**